soundcard/pulseaudio.py

```diff
@@ -13,8 +13,12 @@
 with open(os.path.join(_package_dir, 'pulseaudio.py.h'), 'rt') as f:
     _ffi.cdef(f.read())
 
-_pa = _ffi.dlopen('pulse')
-
+try:
+    _pa = _ffi.dlopen('pulse')
+except:
```

Comment:
Please only catch the correct exception

Reply:
see [ commit 0135d95](https://github.com/bastibe/SoundCard/pull/139/commits/0135d9545f5a63f6b35027612636b37d0e109276)